TrainingExtensions/common/src/python/aimet_common/utils.py

```diff
@@ -222,7 +222,7 @@ def start_bokeh_server_session(port: int):
 
     host_name = socket.gethostname()
     bokeh_serve_command = "bokeh serve  --allow-websocket-origin=" + \
-                          host_name + ":" + str(port) + " --port=" + str(port) + " &"
```

Comment:
Thank you submitting the fix. Could you please : (1) squash the commits on the PR to a single commit (2) update the commit message to indicate what has been fixed (example commit message: "bokeh fix to enable websocket connections originating from additional hosts in visualization api".

Reply:
Hi  Sangeetha,
I have squashed the commits and added an appropriate commit message as per your suggestions. Please have a look and let me know if anything else should be added.

Reply:
Thank you for the quick updates on this one. Have approved the PR.